rmgpy/rmg/main.py

```diff
@@ -1232,6 +1235,44 @@ def execute(self, initialize=True, **kwargs):
                     os.path.join(self.output_directory, "chemkin", "chem_annotated-gas.inp"),
                     surface_file=(os.path.join(self.output_directory, "chemkin", "chem_annotated-surface.inp")),
                 )
+                if self.thermo_coverage_dependence:
+                    # add thermo coverage dependence to Cantera files
+                    chem_yaml_path = os.path.join(self.output_directory, "cantera", "chem.yaml")
+                    gas = ct.Solution(chem_yaml_path, "gas")
+                    surf = ct.Interface(chem_yaml_path, "surface1", [gas])
```

Comment:
Instead of loading the cantera gas and surf objects, we can get the order of species names in the cantera file with something like this:
`cantera_names = [content["species"][i]['name'] for i in range(len(content["species"]))]`